extensions/positron-python/python_files/positron/pinned-test-requirements.txt

```diff
@@ -1,9 +1,11 @@
-bokeh==3.5.1; python_version >= '3.10'
-bokeh==3.4.3; python_version == '3.9'
+bokeh==3.4.3; python_version >= '3.9'
```

Comment:
Had to mess with bokeh requirements because panel randomly has strict upper limits on versions.